### PR TITLE
Add some annotations for recent events

### PR DESCRIPTION
**Pull request by bradcray**
I focused on those that affected fasta and LCALS for the sake of
tweeting and things I improved.

### CODE REVIEW

**Comment by bradcray** [comment]
@ronawho : Would you review?

File: test/ANNOTATIONS.yaml [hunk 1]
```diff
@@ -487,6 +495,8 @@ LCALSMain:
   10/20/16:
     - Clean up to HYDRO_2D kernel (#4757)
     - update to TRAP_INT serial kernel (#4756)
+  12/09/16:
```
Comment (ronawho) [inline test/ANNOTATIONS.yaml]: should be 12/10/16

**Comment by ronawho** [comment]
Think you have the "commit" dates for most things instead of the "affected" dates: https://github.com/bradcray/chapel/blob/8f730c2c3b0e01c29b4a7101a45f4538a5fa7fce/test/ANNOTATIONS.yaml#L26-L29

File: test/ANNOTATIONS.yaml [hunk 2]
```diff
@@ -207,6 +207,8 @@ all:
     - Stack allocate on and task function argument bundles (#4074)
   12/08/16:
     - Re-enable denormalization (#4547)
+  12/12/16:
```
Comment (lydia-duncan) [inline test/ANNOTATIONS.yaml]: I think this one should be 12/13/16 instead.  Multilocale testing worked that night and so the change turns up there, and there was a different set of tests impacted by commits from the 13th.

At least, that's only if I'm right in remember that we enabled annotations on days there isn't data (which I thought I saw go by a while ago, but I may be misremembering)

Comment (lydia-duncan) [inline test/ANNOTATIONS.yaml]: Oh, nevermind about the 13th, there's a different commit for those (I just verified that the coforall one went in on the 13th)

File: test/ANNOTATIONS.yaml (hunk 1 above)
Comment (bradcray) [inline test/ANNOTATIONS.yaml]: Sigh...

**Pull request merged**